drivers/cc1200/gnrc_cc1200/gnrc_netdev_cc1200.c

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
Please have a look at #7414 and the related `netif` rework

Reply:
As I wanted to make the driver compliant to the 802.15.4 Layer (the CC1200 actually has hardware support for that, which I have not enabled by default, as I fear it would collide with RIOT) I don`t know how necessary this file is. Right now the driver does not support RAW packages and is only able to send packets through gnrc_netf_ieee802154. As I may be very well wrong with my approach, could you comment on that?

Reply:
@AnonMall  I can not comment on the 802.15.4  hardware capabilities of this device for right. As for the *gnrc_netdev_cc1200.c* file I would also like to get rid of it and go with a [`gnrc_netif`](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/netif/gnrc_netif.c) + [`gnrc_netif_ieee802154`](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/netif/gnrc_netif_ieee802154.c) solution. For the initialization of 802.15.4 or RAW interfaces it might also be interesting to see how this is currently done for the [Atmel radio](https://github.com/RIOT-OS/RIOT/blob/master/sys/auto_init/netif/auto_init_at86rf2xx.c) (you you haven't done so already).